templates/home.html

```diff
@@ -51,14 +51,8 @@ <h3 class="mb-4">Welcome<span v-cloak v-if="username"> back</span> to Wikipedia
 
         <div class="alert alert-warning" v-cloak role="alert" v-html="streakText" v-if="loggedIn && streakText"></div>
 
-        <p>The rules of this game are simple. You'll be given a starting Wikipedia article and an ending Wikipedia article. Your goal is to navigate from the starting article to the ending article using only the article links.</p>
-        <p>Some other things to keep in mind:</p>
-        <ul>
-            <li>Any link (outside the navigation bar at the top) is fair game.</li>
-            <li>You can't use the browser find tool.</li>
-            <li>Reloading the page or clicking the browser back button will end your run.</li>
-        </ul>
-        <p>Remember, luck is king, so put on your lucky hat, grab your lucky socks, and let the games begin!</p>
+        <p>🏁 <b>Explore the world's largest encyclopedia with lightning speed!</b> Navigate from a starting Wikipedia article to the target using only links shown in each page as fast as you can. </p>
```

Comment:
Hmm I think this is still a little overcomplicated, I think

"Start from one Wikipedia page and navigate to the goal page as fast as you can!"

1) target --> goal is clearer I think
2) Simpler is better + the part about clicking links is kind of confusing, so it does more harm than good imo. It will be obvious how to play for most people, and for those who are confused we have the tutorial that is very explicit.

Reply:
Agreed, I'll add in this change